src/npm.rs

```diff
@@ -0,0 +1,32 @@
+use emoji;
+use std::process::Command;
+
+pub fn npm_pack(path: &str) {
```

Comment:
Can we refactor this into a single method? Do you foresee that it will be changed at any time?

Reply:
Yeah, there are lots of other options you can pass to publish in the future

Reply:
if they are just adding new arguments update, I would suggest to move them to a single function with `struct`

ignore if it is not.

Reply:
dont think its going to be that easy https://docs.npmjs.com/cli/publish

Reply:
i would recommend not moving them into a single struct. i much prefer the way it is set up now!